### PR TITLE
added link to thefeedbackloop.xyz articles for explaning stage-x

### LINKED ISSUE #1082: Link to thefeedbackloop.xyz articles for explaning stage-x
To modify: https://babeljs.io/docs/plugins/#presets-stage-x-experimental-presets-

by editing https://github.com/babel/babel.github.io/blob/master/docs/plugins.md

screen shot 2016-12-13 at 2 28 25 pm

- https://thefeedbackloop.xyz/tc39-a-process-sketch-stages-0-and-1/
- https://thefeedbackloop.xyz/tc39-process-sketch-stage-2/
- [ ] when it's out

Figure out a good place to link to the articles for more of an explanation and/or changing our wording to reflect it better

### PR DESCRIPTION
**Pull request by nggolu**
cc @hzoo 
fixes #1082

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ Any transforms in stage-x presets are changes to the language that haven't been
 The [TC39](https://github.com/tc39) categorizes proposals into the following stages:
 
 - [stage-0](/docs/plugins/preset-stage-0/) - Strawman: just an idea, possible Babel plugin.
-- [stage-1](/docs/plugins/preset-stage-1/) - Proposal: this is worth working on.
-- [stage-2](/docs/plugins/preset-stage-2/) - Draft: initial spec.
+- [stage-1](/docs/plugins/preset-stage-1/) - Proposal: this is worth working on.[Article on Stage-0 and stage--1:](https://thefeedbackloop.xyz/tc39-a-process-sketch-stages-0-and-1/)
```
Comment (hzoo) [inline docs/plugins.md]: Cool can you add a space for both the links? and `stage--1` to be `Stage-1`?

**Comment by hzoo** [comment]
screen shot 2017-04-03 at 8 55 43 am

if you check the rendered state the spacing and capitalization is off (this is what I was referring to in the comment)

File: docs/plugins.md [hunk 2]
```diff
@@ -55,8 +55,8 @@ Any transforms in stage-x presets are changes to the language that haven't been
 The [TC39](https://github.com/tc39) categorizes proposals into the following stages:
 
 - [stage-0](/docs/plugins/preset-stage-0/) - Strawman: just an idea, possible Babel plugin.
-- [stage-1](/docs/plugins/preset-stage-1/) - Proposal: this is worth working on.
-- [stage-2](/docs/plugins/preset-stage-2/) - Draft: initial spec.
+- [stage-1](/docs/plugins/preset-stage-1/) - Proposal: this is worth working on. [Article on Stage-0 and Stage-1:](https://thefeedbackloop.xyz/tc39-a-process-sketch-stages-0-and-1/)
```
Comment (xtuc) [inline docs/plugins.md]: You can remove the `:` since it will be transformed into a link using markdown.

File: docs/plugins.md [hunk 3]
```diff
@@ -55,8 +55,8 @@ Any transforms in stage-x presets are changes to the language that haven't been
 The [TC39](https://github.com/tc39) categorizes proposals into the following stages:
 
 - [stage-0](/docs/plugins/preset-stage-0/) - Strawman: just an idea, possible Babel plugin.
-- [stage-1](/docs/plugins/preset-stage-1/) - Proposal: this is worth working on.
-- [stage-2](/docs/plugins/preset-stage-2/) - Draft: initial spec.
+- [stage-1](/docs/plugins/preset-stage-1/) - Proposal: this is worth working on. [Article on Stage-0 and Stage-1](https://thefeedbackloop.xyz/tc39-a-process-sketch-stages-0-and-1/)
```
Comment (aaronang) [inline docs/plugins.md]: For consistency I would also prefer to end the sentence with a period.

**Comment by existentialism** [comment]
I find that the links now read awkwardly:

image

What about:

> The TC39 categorizes proposals into the following stages:
> 
> - Stage 0 - Strawman: just an idea, possible Babel plugin.
> - Stage 1 - Proposal: this is worth working on.
> - Stage 2 - Draft: initial spec.
> - Stage 3 - Candidate: complete spec and initial browser implementations.
> - Stage 4 - Finished: will be added to the next yearly release.
> 
> For more information, be sure to check out the [current TC39 proposals](https://github.com/tc39/proposals) and its [process document](https://tc39.github.io/process-document).
> 
> The TC39 stage process is also detailed across a few posts over at [thefeedbackloop.xyz](https://thefeedbackloop.xyz/) ([Stage 0 and 1](https://thefeedbackloop.xyz/tc39-a-process-sketch-stages-0-and-1/), [Stage 2](https://thefeedbackloop.xyz/tc39-process-sketch-stage-2/), [Stage 3](https://thefeedbackloop.xyz/tc39-process-sketch-stage-3/), Stage 4 coming soon).

**Comment by existentialism** [comment]
@nggolu thanks!

**Comment by nggolu** [comment]
@existentialism its okay :)
BTW my slack nickname  : goku

**Pull request merged**